Examples/UIExplorer/UIExplorer/RNUserDefaultsIOS.h

```diff
@@ -0,0 +1,10 @@
+//
+// Extracted from https://github.com/dsibiski/react-native-userdefaults-ios
```

Comment:
True, it's best to use `RCTSettings` for this simple use case. My lib will end up being used for those who need to use custom suites.